src/transport/http2.rs

```diff
@@ -299,13 +299,21 @@ fn spawn_recv_forwarder(req: Body, req_tx: Sender<hyper::Result<Bytes>>) -> Join
                     false
                 }
                 Err(cause) => {
-                    error!("Network error: {}", cause);
+                    // Indicates that the connection has been closed on the client side.
+                    // This is a normal occurrence, e.g. when the client has raced the RPC
+                    // call with something else and has droppped the future.
+                    debug!("Network error: {}", cause);
```

Comment:
Are we propagating this dropping?  Will the computation we're running to handle this request be dropped as well?

Not for this PR, but maybe a TODO to leave if there's more work to do this.

Reply:
I think we are, indirectly. This tokio task is what keeps the flume sender alive. If we exit the task, we drop the flume::Sender, which will cause the flume::Receiver to produce an error.

Reply:
yeah, but the flume receiver will only error if you poll it again.  and it probably has already pulled off the request and is processing it?